reactioncommerce/reaction
# 3356 Fix sms TypeError: Cannot read property 'profile' of undefined

resolve #3356

This PR fix a TypeError of profile of undefined. This actually happens whenever send sms is triggered which doesn't find a user in the database thus throwing the profile of undefined error.

#### Fix
Checked that the user exist before calling the profile from the user object else return false.

#### Test
Configure email, shipping, Stripe payment

- Place an order as a user
- go to any mongo client(Robo Mongo) 
- In the Account collection delete the user
- Login as an Admin
- Go to Orders
-  Fulfil order
- Click on Approve then Capture
- Scroll down Click of `all Items packed`
- Observe the green toastr appear sending an email
- Observe server console to see no error.
